pkg/cmd/cli/install/install.go

```diff
@@ -86,13 +86,16 @@ func NewInstallOptions() *InstallOptions {
 
 // AsVeleroOptions translates the values provided at the command line into values used to instantiate Kubernetes resources
 func (o *InstallOptions) AsVeleroOptions() (*install.VeleroOptions, error) {
-	realPath, err := filepath.Abs(o.SecretFile)
-	if err != nil {
-		return nil, err
-	}
-	secretData, err := ioutil.ReadFile(realPath)
-	if err != nil {
-		return nil, err
+	var secretData []byte
+	if o.SecretFile != "" {
```

Comment:
I wonder if it would make sense to try and reduce the possibility of this value not being there by mistake by only making secret optional if the provider is `aws`. WDYT?

Reply:
Currently our only documented use case is AWS, so this is a good point. However, GCP and Azure both have something similar to instance roles/profiles which could theoretically work for Velero, they're just not documented right now.

I've gone back and forth on making a `--no-secret` flag and requiring either that or `--secret-file`, so that it's explicit that the user doesn't want to create a secret rather than simply forgetting the flag. WDYT about that option?

Reply:
Another option would be an interactive confirmation prompt ("Are you sure you don't want to create a credentials secret?").

Reply:
Also true. But I'd think we wouldn't want to completely close off scripting, so it seems to me like we'd have to do both in that scenario...or make them pipe something to stdin.

Reply:
I like the idea of an explicit `--no-secret` flag

Reply:
Yeah..going with just the flag seems reasonable to me.

Reply:
I'd rather not have any prompt during the install. I would not at all mind the one flag OR the other as stated ☝️ and make sure the user knows this upfront - 👍

Reply:
Added `--no-secret` in latest commit.